lib/timex.ex

```diff
@@ -771,26 +771,33 @@ defmodule Timex do
   and third.
 
   By default, the `start`and `ending` bounds are *exclusive*. You can opt for inclusive bounds
-  by setting the `:inclusive` option to `true`.
+  by setting the `:inclusive` option to `true`. You can also specify `:inclusive_start` and/or
+  `:inclusive_end` options, which will override the `:inclusive` option for the given bound.
 
   """
   @type between_options :: [
-    inclusive: boolean
+    inclusive: boolean,
+    inclusive_start: boolean,
+    inclusive_end: boolean
```

Comment:
it might be nicer to expand the domain of `inclusive` rather than introducing new options. for example:

```
iex> Timex.between?(t1, t2, t3, inclusive: true)
iex> Timex.between?(t1, t2, t3, inclusive: :start)
iex> Timex.between?(t1, t2, t3, inclusive: :end)
```
that would avoid some awkward potential situations where `inclusive` and `inclusive_start` and `inclusive_end` disagree with one another.

```
iex> Timex.between?(t1, t2, t3, inclusive: false, inclusive_start: true)
iex> Timex.between?(t1, t2, t3, inclusive: true, inclusive_start: false)
```